assets/thumbnails/thumbnail_howto_use_template.asciidoc

```diff
@@ -0,0 +1,3 @@
+== How to use the template for creating thumbnails
```

Comment:
I think that you could wait to check in this file until you actually have the description.

Reply:
Thanks for your hint. I removed the file and will check in later when finished. Description in the template should be working for now. :-)